modules/ml/src/test/java/org/apache/ignite/ml/tree/randomforest/RandomForestRegressionTrainerTest.java

```diff
@@ -72,16 +71,14 @@ public void testFit() {
             sample.put(x1 * x2 + x3 * x4, new double[] {x1, x2, x3, x4});
         }
 
-        RandomForestRegressionTrainer trainer = new RandomForestRegressionTrainer(4, 3, 5, 0.3, 4, 0.1)
-            .withUseIndex(false);
+        ArrayList<FeatureMeta> meta = new ArrayList<>();
+        for(int i = 0; i < 4; i++)
+            meta.add(new FeatureMeta(i, false));
+        RandomForestRegressionTrainer trainer = new RandomForestRegressionTrainer(meta)
+            .withCountOfTrees(5)
```

Comment:
What about to add this numbers as parameters and performs testing with different values?